src/elements/embed/EmbedSpec.tsx

```diff
@@ -21,6 +22,10 @@ export type MainEmbedProps = {
   createCaptionPlugins?: (schema: Schema) => Plugin[];
 };
 
+const isCallout = (html: string) => {
+  return html.includes("data-callout-tagname");
```

Comment:
It's a shame we don't have anything else on the element data we can use to determine this!

Reply:
It is a bit rubbish, though consistent with how `flexible-content` determines callouts in the old element.

A minor improvement might be checking that the html is valid, and the parent node contains a property called "data-callout-name" which has an actual value. For now I think I'd go with consistency with the old element though.

Reply:
That might be a good way to do it, we could memoise the function and extract the `data-callout-name` value, it also relates to my other comment about the current tag extraction.

Reply:
I'm now getting the tag up front in `EmbedSpec` and passing only the tag into the Callout, since that's all it really needs. `extractTag` is gone - the functionality is part of `getCalloutTag` (replacing `isCallout`)

Reply:
This looks great, I would recommend we [memoise it](https://reactjs.org/docs/hooks-reference.html#usememo) as a performance enhancement.

Reply:
(Sam and I paired on this and decided only to memoise the network request-producing call to Targeting)